package.json

```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "build": "browserify ./js/math.js -o ./js/bundle.js",
-    "test": "jest"
+    "test": "jest",
+    "lint": "eslint ./js",
+    "lint:fix": "eslint ./js --fix"
```

Comment:
is it added to IDE?

Reply:
Yep.
![image](https://user-images.githubusercontent.com/15876570/118754581-9d296000-b824-11eb-8427-50bb416c488a.png)